setup.py

```diff
@@ -67,7 +67,13 @@
 
 install_requires = [
     'Flask>=0.11.1',
-    'requests>=2.4.0',
+    # elasticsearch has hard version range dependency on requests and urllib3
+    # requests has hard version range dependency on idna
+    # Every time idna and urllib3 are updated, installation breaks because
+    # elasticsearch and requests dependencies are not resolved properly.
+    'requests>=2.4.0,<3.0.0', # from elasticsearch
+    'urllib3<1.23,>=1.21.1',  # from elasticsearch
+    'idna<2.7.0', # from requests
```

Comment:
The latest `requests` specifies [<2.8.0](https://github.com/requests/requests/blob/991e8b76b7a9d21f698b24fa0058d3d5968721bc/setup.py#L53).